longevity_mv_synchronous_updates_test.py

```diff
@@ -0,0 +1,60 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+# This is stress longevity test that runs light weight transactions in parallel with different node operations:
+# disruptive and not disruptive
+#
+# After the test is finished will be performed the data validation.
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
+
+from longevity_test import LongevityTest
+from sdcm.utils.common import keyspace_min_max_tokens
+from sdcm.sct_events.group_common_events import ignore_mutation_write_errors
+
+
+class MvSynchronousLongevityTest(LongevityTest):
+
+    def run_prepare_write_cmd(self):
+        with ignore_mutation_write_errors():
+            super().run_prepare_write_cmd()
+
+        self.log.debug("Start create_main_mv")
+        self.create_main_mv()
+        self.log.debug("Finish create_main_mv")
```

Comment:
I think that this entire thing is not necessary.
IIUC, you wanted to create the MV after you make sure you ran the prepare, but I think it would be easier and clearer to run "pre_create_schema" for the main table and/or also for the MVs.

Reply:
"pre_create_schema" use pre-defined query. I need to get token ranges t build this query for MV.
I can use "pre_create_schema" mechanism but still need to overwrite this function

Reply:
I read this and I don't understand why we can't use token also as part of the user profile.
Only because you don't know the view_token_min and view_token_max ?

You can decide whatever you want and hard code them in the user profile.